server/routes/victorops/tasks.py

```
@@ -0,0 +1,427 @@
+            try:
+                return int(raw)
+            except (ValueError, TypeError):
+                pass
+    return None
+
+
+# ---------------------------------------------------------------------------
+# Processing helpers (extracted to keep task function complexity manageable)
+# ---------------------------------------------------------------------------
+
+def _record_primary_alert(cursor, conn, user_id, org_id, incident_db_id, event_db_id,
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_record_primary_alert`

Add return type annotation: `None`

(ANN202)